clusterman/interfaces/cluster_connector.py

```diff
@@ -35,6 +37,10 @@ def reload_state(self) -> None:  # pragma: no cover
         """ Refresh any state that needs to be stored at the start of an autoscaling run """
         pass
 
+    def get_removed_nodes_since_last_reload(self) -> Set[KubernetesNode]:
```

Comment:
I am on the fence about whether this should be included here if it's only supported for one subclass.  I get that it's here to pass mypy presumably, but maybe there's another way to do it that's cleaner?

Not too sure, if there's not a better way to do this, feel free to leave it alone.

Reply:
It looks like mypy is happy without it, removed